agent/command/git_push.go

```diff
@@ -163,15 +163,14 @@ func (c *gitPush) pushPatch(ctx context.Context, logger client.LoggerProducer, p
 	cmd := jpm.CreateCommand(ctx).Directory(p.directory).Append(pushCommand).
 		SetOutputSender(level.Info, logger.Task().GetSender()).SetErrorWriter(stdErr)
 	if err := cmd.Run(ctx); err != nil {
+		errorOutput := stdErr.String()
```

Comment:
looking at this again, maybe we could log stderr before we check the error? Something like 
```
err := cmd.Run(ctx)
[log stderr...]
if err != nil {
    return errors.Wrap(err, "can't push to remote")
}
```
this way stderr would get logged regardless of the exit code. 
I'm not certain if git would print things on stderr if we're exiting 0, but maybe we should handle it?

Reply:
sure, I'm neutral